src/main/java/org/mdbenefits/app/utils/ApplicationUtilities.java

```diff
@@ -0,0 +1,29 @@
+package org.mdbenefits.app.utils;
+
+import formflow.library.data.Submission;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.bouncycastle.util.Strings;
+import org.mdbenefits.app.data.enums.HelpNeededType;
+import org.springframework.shell.Command.Help;
+
+public class ApplicationUtilities {
+
+        public static Set<String> flattenHouseholdNeedsPrograms(ArrayList<String> needTypes) {
+                ArrayList<ArrayList<String>> relevantPrograms = new ArrayList<>();
```

Comment:
I think you could just do `List<List<String>>` here for the left side of the equals.